packages/vite-plugin/src/index.ts

```diff
@@ -183,10 +186,16 @@ export function vanillaExtractPlugin({
             cssMap.get(absoluteId) !== source
           ) {
             const { moduleGraph } = server;
-            const module = moduleGraph.getModuleById(absoluteId);
+            const [module] = Array.from(
+              moduleGraph.getModulesByFile(absoluteId) || [],
```

Comment:
Not saying this is wrong, just for my own knowledge would like to understand this change more? 

Assuming the `absoluteId` doesn't match up with the vite id but does match a file?

Reply:
> Assuming the absoluteId doesn't match up with the vite id but does match a file?

That's right, according to my tests. A "module id" consists of a file path plus an optional query string that Vite adds for HMR or for other reasons. However, our `absoluteId` variable here is always created without a query string, so it sometimes doesn't match the module id. It always matches the file path, though.